src/cow-react/modules/limitOrders/hooks/useTradeFlowContext.ts

```diff
@@ -0,0 +1,69 @@
+import { TradeFlowContext } from 'cow-react/modules/limitOrders/services/tradeFlow'
+import { useWeb3React } from '@web3-react/core'
+import { OrderKind } from '@cowprotocol/contracts'
+import { useLimitOrdersTradeState } from 'cow-react/modules/limitOrders/hooks/useLimitOrdersTradeState'
+import { CurrencyAmount, Percent } from '@uniswap/sdk-core'
+import { Token } from '@uniswap/sdk-core'
+import { useWalletInfo } from 'hooks/useWalletInfo'
+import { useGP2SettlementContract } from 'hooks/useContract'
+import { useDispatch } from 'react-redux'
+import { AppDispatch } from 'state'
+import { useAppData } from 'hooks/useAppData'
+
+// TODO: set proper values
+const deadline = 5 // 5 min
+const allowedSlippage = new Percent(0)
+const recipientAddressOrName = null
+const feeAmountRawValue = 60000000
```

Comment:
This might be `0` as well but I know this is a bogus value for now

Reply:
indeed, you can set already this to zero. It should always be zero for limit orders

Reply:
https://github.com/cowprotocol/cowswap/pull/1124
In this PR I have added requesting of quote and get a fee amount from it